docs/components/demo/Drawer/css/styles.css

```
@@ -24,12 +24,19 @@ input {
 }
 
 .DrawerContent {
+  /* Extra drawer area that hangs below the viewport edge, kept off-screen by an
+     equal negative margin. Pulling the drawer away from its edge slides the
+     bleed into view instead of exposing the overlay behind it, so the drawer
+     stretches rather than visually detaching while it rubber-bands. */
+  --bleed: 48px;
   background-color: white;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)